### ISSUE TITLE
Feature request: generate sentence from given input word

### ISSUE THREAD

**Issue by TristanJM** [jsvine/markovify#91]
Does the API provide a way to generate a sentence that contains the `input word`?

eg. 
Input word: accessed
Output: He went to the bank because somebody had accessed his account.

If the lib doesn't currently support this, but it can be achieved with simple modification, feel free to point me in the right direction and I'll have a go.

**Comment by jsvine** [comment]
HI Tristan, and interesting question! That feature does not currently exist, largely because of the way Markov chains work (i.e., starting with a known state and generating subsequent states based on observed probabilities).

If you want to implement this for your own projects, the simplest / brute-force-iest approach is probably just to generate a bunch of sentences and keep only those that contain your desired word.

A more efficient approach (but more complex to code) might be to filter the model's dictionary of possible states, and simultaneously work backwards and forwards from your desired word.

**Comment by OtakuMegane** [comment]
The "brute-force" approach is what I had to settle on a while back. It works but obviously not very efficient and is prone to some issues of balancing attempts vs processing time.

The one case I know that actually builds around a given word is the old [Markovsky IRC](https://github.com/Mego/Markovsky). As far as I can interpret, it does implement the idea of building both backwards and forwards from the word. I pulled the specific function out into a paste in case the actual code helps (it's C++): https://pastebin.com/wXNkt8ne

I have from time to time considered trying to make this work with markovify but I still don't understand the inner workings enough to do it.
